run/events-pubsub/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# [START run_events_pubsub_dockerfile]
+# Use the official lightweight Python image.
+# https://hub.docker.com/_/python
+FROM python:3.7-slim
+
+# Copy local code to the container image.
+ENV APP_HOME /app
+ENV PORT 8080
+WORKDIR $APP_HOME
+COPY . ./
+
+# Install production dependencies.
+RUN pip install Flask gunicorn
+
+# Run the web service on container startup. Here we use the gunicorn
+# webserver, with one worker process and 8 threads.
+# For environments with multiple CPU cores, increase the number of workers
+# to be equal to the cores available.
+CMD exec gunicorn --bind :$PORT --workers 1 --threads 8 app:app
```

Comment:
Needs timeout flag to avoid "CRITICAL WORKER TIMEOUT".  See full Dockerfile example: https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/run/pubsub/Dockerfile